ocpmodels/trainers/forces_trainer.py

```diff
@@ -82,36 +96,56 @@ def load_task(self):
         num_targets = 1
 
         self.num_targets = num_targets
-        (
-            self.train_loader,
-            self.val_loader,
-            self.test_loader,
-        ) = self.dataset.get_dataloaders(
-            batch_size=self.config["optim"]["batch_size"]
+        # (
+        #     self.train_loader,
+        #     self.val_loader,
+        #     self.test_loader,
+        # ) = self.dataset.get_dataloaders(
+        #     batch_size=self.config["optim"]["batch_size"]
+        # )
+        self.train_loader = DataLoader(
+            self.dataset,
+            batch_size=5,
+            shuffle=False,
+            collate_fn=collater,
+            num_workers=16,
         )
+        self.val_loader, self.test_loader = None, None
 
         # Normalizer for the dataset.
         # Compute mean, std of training set labels.
         self.normalizers = {}
         if self.config["dataset"].get("normalize_labels", True):
-            self.normalizers["target"] = Normalizer(
-                self.train_loader.dataset.data.y[
-                    self.train_loader.dataset.__indices__
-                ],
+            # self.normalizers["target"] = Normalizer(
+            #     self.train_loader.dataset.data.y[
+            #         self.train_loader.dataset.__indices__
+            #     ],
+            #     self.device,
+            # )
+            self.normalizers["target"] = Normalizer().from_precomputed(
+                self.config["dataset"]["target_mean"],
+                self.config["dataset"]["target_std"],
```

Comment:
This logic requires us to define target_mean and target_std's in all dataset scenarios. Maybe we can condition these two scenarios on the dataset type `traj_folder` vs `trajectory`?

Also, can we make the target_mean and target_std computation part of the model, seems a bit awkward to have it outside?